src/sentry/static/sentry/app/views/settings/projectAlerts/projectAlertRulesNew.tsx

```diff
@@ -81,23 +81,73 @@ class ProjectAlertRules extends AsyncView<
             />
           );
         })}
-      </div>
+
+        {this.state.metricRules.map(rule => {
+          return (
+            <RuleRow
+              type="metric"
+              api={this.api}
+              key={`metric-${rule.id}`}
+              data={rule}
+              orgId={orgId}
+              projectId={projectId}
+              params={this.props.params}
+              location={this.props.location}
+              routes={this.props.routes}
+              onDelete={this.handleDeleteRule.bind(this, rule)}
+              canEdit={canEditRule}
+            />
+          );
+        })}
+      </React.Fragment>
     );
   }
 
+  renderLoading() {
+    return this.renderBody();
+  }
+
   renderBody() {
-    const {ruleList} = this.state;
-    const {projectId} = this.props.params;
+    const {loading, issueRules} = this.state;
 
     return (
       <React.Fragment>
-        <SentryDocumentTitle title={t('Alerts Rules')} objSlug={projectId} />
         <PermissionAlert />
-        {!!ruleList.length && this.renderResults()}
-        {!ruleList.length && this.renderEmpty()}
+
+        <Panel>
+          <RuleHeader>
+            <div>{t('Type')}</div>
+            <div>{t('Name')}</div>
+            <TriggerAndActions>
+              <div>{t('Conditions/Triggers')}</div>
+              <div>{t('Action(s)')}</div>
+            </TriggerAndActions>
+          </RuleHeader>
+
+          <PanelBody>
+            {loading
+              ? super.renderLoading()
+              : !!issueRules.length
+              ? this.renderResults()
+              : this.renderEmpty()}
+          </PanelBody>
+        </Panel>
       </React.Fragment>
     );
   }
 }
 
-export default ProjectAlertRules;
+export default withOrganization(ProjectAlertRules);
+
+const RuleHeader = styled(PanelHeader)`
```

Comment:
Will probably follow-up to refactor this CSS into a common module since it needs to be shared.